CONTRIBUTING.md

```diff
@@ -11,7 +11,20 @@ Please fill out either the individual or corporate [Contributor License Agreemen
 
 Follow either of the two links above to access the appropriate CLA and instructions for how to sign and return it. Once we receive it, we'll be able to accept your pull requests.
 
-## Contributing A Patch
+## How to Build and Test
+
+Look into `./test.sh` to understand how. Minimally,
+
+1. Build `dpkg_parser.par` first, if not done so: `bazel build //package_manager:dpkg_parser.par` (You may need to provide `--host_force_python=PY2` as in `test.sh`.)
+
+   You don't have to repeat this step unless you cleaned your workspace or want to generate a new version of `dpkg_parser.par`.
+1. `bazel build //...`
```

Comment:
```suggestion
2. `bazel build //...`
```

Reply:
This will become "2"  when rendered, although just saying "2" works. But since other parts of the doc uses this pattern, I'll keep it as "1".